idl/storage_service.idl.hh

```diff
@@ -10,10 +10,12 @@ namespace service {
   struct raft_topology_cmd {
       enum class command: uint8_t {
           barrier,
+          barrier_and_drain,
```

Comment:
Breaks ABI compatibility. Add new enums at the end.

Must protect with a cluster feature to avoid use before it's ready.

Or perhaps we can sweep all this under the carpet since it's still experimental.

Reply:
It is experimental. We can change it freely at this point.